skrifa/src/outline/glyf/hint/graphics_state/mod.rs

```diff
@@ -104,6 +104,10 @@ pub struct GraphicsState<'a> {
     ///
```

Comment:
`backward_compatibility` is a bit hard to follow. Despite the comment and the freetype link I'm still not entirely clear what we're being backward compatible with. If possible expand the comment and/or rename the field.

Reply:
I’ll add an extended description based on my explanation above. I think there’s a useful MS resource about this as well. I’ll try to find a link.